templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -226,8 +226,8 @@ contents:
         # br-ex has a different mac address than the bond which might be
         # troublesome on some platforms where the nic won't accept packets with
         # a different destination mac. But nobody has complained so far so go on
-        # with what we got. 
-        
+        # with what we got.
```

Comment:
Remove all the tabs? :)

Reply:
it was automatically added :( tried to remove them and failed as goland just adds them